### DISCUSSION TITLE
Error if i compile coverletter.tex

### DISCUSSION THREAD

**Discussion by ritayadav** [posquit0/Awesome-CV#604] [General]
![error](https://cloud.githubusercontent.com/assets/21067142/22012143/c01f866a-dcba-11e6-93eb-dac3d30ffe79.png)

I have already profile in examples folder, still i get error, please check in screenshot.

**Comment by posquit0** [comment]
Did you compile with `make` command?
If you tried to compile `coverletter.tex` in `examples` directory, edit the path for profile image to `./profile`.

**Comment by ritayadav** [comment]
I got the solution, I just changed the photo path to "../examples/profile"
